convert_old_alw_to_new_structure.py

```diff
@@ -0,0 +1,26 @@
+import csv
+#csv file from which the original data is read
+with open('hr.contract (8).csv') as csvfile1: 
+	#csv file to which new data is written
+	with open('csvfile.csv','w') as csvfile2:
+		#the keys in the new csv file
+		fieldnames = ['External ID', 'Allowances', 'Allowances/Amount']
```

Comment:
Please do not hard code fieldnames, this can easily be read for the the csv.DictReader instance e.g.
```python
reader = csv.DictReader(x) 
fieldnames = reader.fieldnames
```